ui/app/AppLayouts/Profile/views/ContactsView.qml

```diff
@@ -117,10 +117,10 @@ SettingsContentBase {
                 ContactsListPanel {
                     id: verifiedContacts
                     width: parent.width
-                    height: ((contactsListHeight < (stackLayout.height/2)) ? contactsListHeight :
-                            (stackLayout.height-mutualContacts.contactsListHeight))
-                    scrollbarOn: mutualContacts.contactsListHeight > (stackLayout.height/2) ?
-                                 (contactsListHeight > (stackLayout.height/2)) : (contactsListHeight > parent.height)
+                    height: ((contactsListHeight < (root.height/2)) ? contactsListHeight :
+                                                                      (root.height-mutualContacts.contactsListHeight))
```

Comment:
:eyes: Conditions like this is qml code smell..

Could you please try to use `ColumnLayout` instead of `Column` for items positioning and height management? `Layout.fillHeight: true` should do the trick.